com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -87,6 +87,7 @@ public class SceneEvent
     public class NetworkSceneManager : IDisposable
     {
         private const NetworkDelivery k_DeliveryType = NetworkDelivery.ReliableFragmentedSequenced;
+        internal const int InvalidSceneNameOrPath = -1;
```

Comment:
nit:
```suggestion
        internal const int InvalidSceneNameOrPathHash = -1;
```

Reply:
This is for SceneUtility.GetBuildIndexByScenePath that will return -1 if the scene name or scene path is invalid.